api/ipfsproxy/config.go

```diff
@@ -250,8 +258,14 @@ func (cfg *Config) toJSONConfig() (jcfg *jsonConfig, err error) {
 	jcfg.NodeHTTPS = cfg.NodeHTTPS
 
 	jcfg.ExtractHeadersExtra = cfg.ExtractHeadersExtra
-	jcfg.ExtractHeadersPath = cfg.ExtractHeadersPath
-	if !(cfg.ExtractHeadersTTL > 0) {
+	if cfg.ExtractHeadersPath == DefaultExtractHeadersPath {
```

Comment:
The first case is not necessary because the json object is empty. So the `if` should only have the second case.